rotkehlchen/tests/unit/decoders/test_aerodrome.py

```diff
@@ -136,14 +138,16 @@ def test_add_liquidity(base_transaction_decoder, base_accounts):
 
 
 @pytest.mark.vcr()
+@pytest.mark.parametrize('load_global_caches', [['velodrome']])
 @pytest.mark.parametrize('base_accounts', [['0x514c4BA193c698100DdC998F17F24bDF59c7b6fB']])
-def test_stake_lp_token_to_gauge(base_accounts, base_transaction_decoder):
+def test_stake_lp_token_to_gauge(base_accounts, base_transaction_decoder, load_global_caches):
     evmhash = deserialize_evm_tx_hash('0x9a0cd1ab0b8e5dbf2718b1c87dad239f7f3a9ed8ff2e07643922b190f80ae898 ')  # noqa: E501
     user_address = base_accounts[0]
     events, _ = get_decoded_events_of_transaction(
         evm_inquirer=base_transaction_decoder.evm_inquirer,
         database=base_transaction_decoder.database,
         tx_hash=evmhash,
+        load_global_caches=load_global_caches,
```

Comment:
question: we need to pass load_global_cache here because we don't use the `[chain_name]_transaction_decoder` and instead we create a new one, instead of using the one that you have also modified to have the patch... right?

if instead of `evm_inquirer` and `database` we provide as argument  `[chain_name]_transaction_decoder` to  get_decoded_events_of_transaction we can avoid building it there and having to patch it... also we don't need to care about passing `load_global_caches`. That requires modifying almost all the tests and I believe we can do it after... what do you think? I would add a todo

Reply:
I mention it because having `load_global_caches` is not really doing nothing but defining a constant

Reply:
yeah this makes sense